examples/pipelines/commoncrawl/components/image_url_deduplication/fondant_component.yaml

```diff
@@ -0,0 +1,9 @@
+name: Deduplicate images based on image source url
+description: Component that deduplicate images based on source url
+image: ghcr.io/ml6team/image_url_deduplication:latest
+
+consumes:
```

Comment:
Since #244 is still open, we should consume and produce all the fields that we need

Reply:
Nvm, this is only filtering.